File: ruby/near_hundred.rb
def near_hundred(n)
	return ((n-100).abs <= 10) || (n-200).abs <= 10
end
puts near_hundred(93) ? "pass" : "fail"
puts near_hundred(80) ? "fail" : "pass"
puts near_hundred(89) ? "fail" : "pass"
puts near_hundred(198) ? "pass" : "fail"
puts near_hundred(230) ? "fail" : "pass"
puts near_hundred(202) ? "pass" : "fail"

File: php/near_hundred.php
<?php
	function near_hundred($n)
	{
		return (abs($n-100)<=10  || abs($n-200)<=10);
	}

	echo near_hundred(93) ? "pass" : "fail"; echo "\n";
	echo near_hundred(80) ? "fail" : "pass"; echo "\n";
	echo near_hundred(198) ? "pass" : "fail"; echo "\n";
	echo near_hundred(230) ? "fail" : "pass"; echo "\n";
	echo near_hundred(202) ? "pass" : "fail"; echo "\n";

?>